### ISSUE TITLE
secondary_ranges doesn't work

### ISSUE THREAD

**Issue by electropolis** [terraform-google-modules/terraform-google-network#474]
### TL;DR

Running terraform with module `network` doesn't creates any second `secondary_ranges` networks. Only one is created.

### Expected behavior

Should give information about `secondary_ranges` when running `terraform plan`

```
Terraform will perform the following actions:

  # module.network.module.subnets.google_compute_subnetwork.subnetwork["europe-central2/gke-nodes"] will be created
  + resource "google_compute_subnetwork" "subnetwork" {
      + creation_timestamp         = (known after apply)
      + description                = "Nodes subnet for GKE user-managed"
      + external_ipv6_prefix       = (known after apply)
      + fingerprint                = (known after apply)
      + gateway_address            = (known after apply)
      + id                         = (known after apply)
      + ip_cidr_range              = "10.1.255.0/24"
      + ipv6_cidr_range            = (known after apply)
      + name                       = "gke-nodes"
      + network                    = "gke"
      + private_ip_google_access   = (known after apply)
      + private_ipv6_google_access = (known after apply)
      + project                    = "sysdev-dev"
      + purpose                    = (known after apply)
      + region                     = "europe-central2"
      + secondary_ip_range         = [
          + {
              + ip_cidr_range = "10.1.32.0/23"
              + range_name    = "gke-svc"
            },
        ]
      + self_link                  = (known after apply)
      + stack_type                 = (known after apply)
    }

  # module.network.module.vpc.google_compute_network.network will be created
  + resource "google_compute_network" "network" {
      + auto_create_subnetworks                   = false
      + delete_default_routes_on_create           = false
      + description                               = "Kubernetes network for Nodes"
      + enable_ula_internal_ipv6                  = false
      + gateway_ipv4                              = (known after apply)
      + id                                        = (known after apply)
      + internal_ipv6_range                       = (known after apply)
      + mtu                                       = 0
      + name                                      = "gke"
      + network_firewall_policy_enforcement_order = "AFTER_CLASSIC_FIREWALL"
      + project                                   = "sysdev-dev"
      + routing_mode                              = "GLOBAL"
      + self_link                                 = (known after apply)
    }

Plan: 2 to add, 0 to change, 0 to destroy.
```

`secondary_ip_range` doesn't show the second range.

### Observed behavior

The observed behaviour is that second range of secondary network isn't created 

### Terraform Configuration

```hcl
Running


module "network" {
  source  = "terraform-google-modules/network/google"
  version = "7.2.0"
  # insert the 3 required variables here
  network_name = "gke"
  project_id   = "sysdev-dev"
  description  = "Kubernetes network for Nodes"

  subnets = [
    {
      subnet_name   = "gke-nodes"
      subnet_ip     = "10.1.255.0/24"
      subnet_region = "europe-central2"
      description   = "Nodes subnet for GKE user-managed"
    }
  ]

  secondary_ranges = {
    gke-nodes = [
      {
        range_name    = "gke-pods"
        ip_cidr_range = "10.1.0.0/19"
      },
    ]

    gke-nodes = [
      {
        range_name    = "gke-svc"
        ip_cidr_range = "10.1.32.0/23"
      }
    ]
  }
}
```

### Terraform Version

```sh
❯ tf version
Terraform v1.5.4
on darwin_arm64
+ provider registry.terraform.io/hashicorp/google v4.76.0
+ provider registry.terraform.io/hashicorp/google-beta v4.76.0
```

### Additional information

 This configuration was taken from documentation [here](https://registry.terraform.io/modules/terraform-google-modules/network/google/latest)
 
Google best practice is to use VPC native cluster where Primary Address Range in Subnet is used for Nodes and InternalLB and secondary IP ranges are use for
* pods
* services 
And it takes two of them
https://cloud.google.com/kubernetes-engine/docs/concepts/alias-ips#cluster_sizing

**Comment by electropolis** [comment]
Ok after studying that code I saw that ranges are in the sam object name
